goconfig_test.go

```diff
@@ -68,3 +68,16 @@ func ExampleParse() {
 	// Danyel Bayraktar
 	// cydrop@gmail.com
 }
+
+func BenchmarkExample(b *testing.B) {
+	gitconfig := "configs/danyel.gitconfig"
+	bytes, err := ioutil.ReadFile(gitconfig)
+	if err != nil {
+		log.Fatalf("Couldn't read file %v\n", gitconfig)
```

Comment:
Is there a rule when to use `log.Fatalf` or `b.Fatalf`?

Reply:
Yep, sry here should be `b.Fatalf`, missed than after copy paste. `b.Fatalf` will print readable output and break other benchmarks.

```
--- FAIL: BenchmarkExample
	/goconfig/goconfig_test.go:78: Couldn't read file configs/danyel.gitconfig
FAIL
exit status 1
```